example/properties_directory/non-scalar/hubbard_parameters.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "hubbard_parameters",
+    "units": "eV",
+    "category": "hubbard_u",
```

Comment:
Can we return to reusable `atomic_properties` per orbital instead of adding "category"?

```json
# Reusable: atomic_properties_per_orbital
{
    "name": "ANY",
    "values":
    [
        {
            "id": 1,
            "atomicOrbital": {
                // reusing the content of "atomicOrbital" here, maybe not all of it
                "orbitalName": "3S",
                "orbitalIndex": 1,
                "principalNumber": 3,
                "angularMomentum": 0,
                "occupation": 2
            },
            "value": 6.3
        },
        {
            "id": 1,
            "orbitalName": "3S",
            "value": 3.3
        },
        {
            "id": 2,
            "orbitalName": "3P",
            "value": 6.3
        }
    ]
}
```

Then `hubbard_u` is a specific implementation of an `atomic_properties_per_orbital`, so is `hubbard_v` and `hubbard_j`?

Reply:
We need to decide. Few things to note:
1. We have only `orbitalName` in common. Not sure if it is worth reusing `atomicOrbital`
2. It is a per site property (same species on different sites can have different U values). In case of `V`, it is no longer per site rather inter-site property (`V Co-3d O-2p 1 2 0.2`), so we are going to have somewhat different schema.
3. Property names are used in [application-flavors/executables/](https://github.com/Exabyte-io/application-flavors/blob/dev/executables/espresso/pw.x.yml) to specify the results. We do not have different flavors for `hp.x`, so how do we specify when to use `hubbard_u` and when `hubbard_v` result? I created category for this (keeping the same `name`), which can be populated by the parser depending on the content of `hp.x` output.